### ISSUE TITLE
[QST] Is it normal for the KerasSequenceLoader to output tensors on CPU?

### ISSUE THREAD

**Issue by kunlunl** [NVIDIA-Merlin/NVTabular#1269]
When I tried https://github.com/NVIDIA-Merlin/NVTabular/blob/main/examples/multi-gpu-movielens/01-03-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb , I found that some tensors in `examples` and `labels` (in `for batch, (examples, labes) in enumerate(train_dataset_tf`) are CPU tensors, is this as expected? Shouldn't all tensors are supposed to be on GPU？
